Modules/Data/SpellHit.lua

```diff
@@ -123,7 +123,7 @@ function _SpellHit:GetSpellHitFromBuffs()
             otherDraeneiInGroup = true
         end
 
-        if spellId == 30708 then
+        if ECS.IsTbc and spellId == 30708 then
```

Comment:
```suggestion
        if spellId == 30708 then
```

This is also part of #354